include/assert.h

```diff
@@ -66,20 +66,20 @@
     }                                     \
   while (0)
 #else
-#  define ASSERT(f)                         \
-  do                                        \
-    {                                       \
-      if (predict_false(!(f)))              \
-        __assert(__FILE__, __LINE__, NULL); \
-    }                                       \
+#  define ASSERT(f)                       \
+  do                                      \
+    {                                     \
+      if (predict_false(!(f)))            \
+        __assert(__FILE__, __LINE__, 0);  \
```

Comment:
Will this not lead to an error that integer is passed where the pointer is expected?

Reply:
it's valid to pass 0 to pointer argument.

Reply:
Ok. I just checked. The parameter values are not checked. I was thinking that `-Wint-conversion` will catch it, but seems it handles only assignment cases (and passing a parameter is a kind of indirect assignment, so strange why this is not caught).